ci-operator/step-registry/ipi/conf/azurestack/ipi-conf-azurestack-commands.sh

```diff
@@ -56,3 +56,17 @@ EOF
 
 echo "${AZURESTACK_ENDPOINT}" >> ${SHARED_DIR}/AZURESTACK_ENDPOINT
 echo "${SUFFIX_ENDPOINT}" >> ${SHARED_DIR}/SUFFIX_ENDPOINT
+APP_ID=$(jq -r .clientId "${SHARED_DIR}/osServicePrincipal.json")
+AAD_CLIENT_SECRET=$(jq -r .clientSecret ${SHARED_DIR}/osServicePrincipal.json)
+TENANT_ID=$(jq -r .tenantId "${SHARED_DIR}/osServicePrincipal.json")
+cat >> "${SHARED_DIR}/azurestack-login-script.sh" << EOF
```

Comment:
As QE added workflow for ipi on azure stack wwt (https://github.com/openshift/release/pull/34523), is it possible to merge https://github.com/openshift/release/pull/34879/files#diff-17dfbec7f1a0003deaf70faf7537a689abee62acabefc2b9abe3a24c51c48999L29-L47 to `azurestack-login-script.sh`? so that it can also work on azure stack wwt cloud.

Reply:
@rna-afk wdyt?

Reply:
Fixed it although I didn't add the region if check and used `LEASED_RESOURCE` instead.